spec/function.dd

```diff
@@ -561,6 +561,8 @@ $(H2 $(LNAME2 property-functions, Property Functions))
     $(LI $(D @property) functions cannot be overloaded with non-$(D @property) functions with the same name.)
     $(LI $(D @property) functions can only have zero, one or two parameters.)
     $(LI $(D @property) functions cannot have variadic parameters.)
+    $(LI Binary assignment operators can be used if the appropriate getter and setter $(D @property) functions
+    exist.)
```

Comment:
(somewhere you should mention (and test in the other PR) the interplay with opBinary overloading)